src/systems/ec/implementations/react/packages/inpage-navigation/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "private": true,
+  "name": "@ecl/ec-react-component-inpage-navigation",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.1.1",
+  "description": "ECL EC React Inpage navigation",
+  "dependencies": {
+    "@ecl/ec-react-component-icon": "^2.1.1"
+  },
+  "devDependencies": {
+    "@ecl/ec-component-link": "^2.1.1",
```

Comment:
do you really need `@ecl/ec-component-link`?

Reply:
i love your rhetorical questions... :D